app/api/images/route.ts

```diff
@@ -37,15 +37,20 @@ export async function POST(request: Request) {
 
   // Create the image record in Xata
   const record = await xata.db.image.create({
-    name: name as string,
-    image: {
-      name: fileName,
-      mediaType: mimeType,
-      // Xata expects a base64 encoded string for the file content
-      base64Content: fileData.toString('base64')
-    }
+    name: name as string
   });
 
+  // Upload the file and attach it to the image record
+  await xata.files.upload(
+    {
+      table: 'image',
+      column: 'image',
+      record: record.id
+    },
+    fileData
+  );
+  await xata.db.image.update(record.id, { image: { mediaType: mimeType, name: fileName } });
```

Comment:
If we didn't care about the file name this call can be removed.

Reply:
Good point. I removed this and the related vars we were pushing to it.